src/dbnode/encoding/proto/round_trip_test.go

```diff
@@ -145,7 +146,8 @@ func TestRoundTrip(t *testing.T) {
 	require.NoError(t, err)
 
 	buff := bytes.NewBuffer(rawBytes)
-	iter := NewIterator(buff, testVLSchema, testEncodingOptions)
+	iter := NewIterator(buff, testEncodingOptions)
```

Comment:
Unrelated to your change, but while you’re here, mind moving the test proto files in this package to `src/dbnode/encoding/proto/testdata`